scripts/fires/wfigs_data.py

```diff
@@ -41,13 +41,18 @@
 
 FLAGS = flags.FLAGS
 
+flags.DEFINE_boolean('read_location_cache', True,
+                     'save location cache to file.')
+
 flags.DEFINE_boolean('save_location_cache', False,
                      'save location cache to file.')
 
 PRE_2022_FIRE_LOCATIONS_URL = "https://services3.arcgis.com/T4QMspbfLg3qTGWY/arcgis/rest/services/Fire_History_Locations_Public/FeatureServer/0/query?where=1%3D1&outFields=InitialLatitude,InitialLongitude,InitialResponseAcres,InitialResponseDateTime,UniqueFireIdentifier,IncidentName,IncidentTypeCategory,IrwinID,FireCauseSpecific,FireCauseGeneral,FireCause,FireDiscoveryDateTime,ContainmentDateTime,ControlDateTime,IsCpxChild,CpxID,DiscoveryAcres,DailyAcres,POOFips,POOState,EstimatedCostToDate,TotalIncidentPersonnel,UniqueFireIdentifier&outSR=4326&orderByFields=FireDiscoveryDateTime&f=json&resultType=standard"
-_OUTPUT = "/cns/jv-d/home/datcom/v3_resolved_mcf/fire/wfigs/"
 _LAT_LNG_CACHE = {}
 _START_YEAR = 2014
+_GCS_PROJECT_ID = "datcom-204919"
+_GCS_BUCKET = "datcom-import-cache"
+_GCS_FILE_PATH = "fires/location_file.json"
```

Comment:
Is the checked-in version on git LFS? if not, maybe we should move it there or remove (given its moving to GCS)?

Reply:
Done